### DISCUSSION TITLE
Getting Error during Installation of IDH Node

### DISCUSSION THREAD

**Discussion by ghost** [Security-Onion-Solutions/securityonion#8040] [Unsupported Versions] [answered]
Getting an Error which says, "could not determine version of Security Onion" while setting up IDH node. One more error which comes is "could not download .tar.gz from manager". I recently upgraded my manager search to 2.3.120 and started the IDH installation using the iso file.  Anything I am missing during the installation of IDH node?

**Comment by dougburks** [comment, marked as answer]
Are you able to manually ssh from the IDH node to the manager as the `soremote` user?

**Reply by ghost** [reply]
Hi @dougburks, The Issue got resolved. There were Unaccepted keys and I accepted those in the manager. The installation got complete and the IDH node got added to the grid of Manager as well.
